frontend/src/sections/Navbar/Navbar.tsx

```diff
@@ -31,15 +31,15 @@ const Drawer = ({
           Home
         </Link>
         <Link
-          href='/allEvents'
-          className={`py-3 ${pathname === '/allEvents' && 'font-semibold'}`}
+          href='/events'
```

Comment:
Lets create another `routes.ts` file in constants and keep all the routes in it, then import and use everywhere. Something like:

```
const EVENT_ROUTES = {
  readOne: '/events',
  readAll: '/events/:eventId',
}

export { EVENT_ROUTES }
```

Reply:
sounds good.